README.md

```diff
@@ -91,4 +91,4 @@ The following table contains the parameters currently supported by the device
 | `xsensmt_euler_period`     | double  |    seconds   | `xsensmt_period`  | No   | Period of querying the Euler Xsens MT* device | The frequency of publishing the information is determined by the device that attaches this one |
 | `xsensmt_position_period`     | double  |    seconds   | `xsensmt_period`  | No   | Period of querying the Position Xsens MT* device | The frequency of publishing the information is determined by the device that attaches this one |
 | `xsensmt_linear_velocity_period`     | double  |    seconds   | `xsensmt_period`  | No   | Period of querying the linear velocity Xsens MT* device | The frequency of publishing the information is determined by the device that attaches this one |
-
+| `resetSensorOrientation` | bool|   | `false`  | No        | Option to be set to true if the orientation has to be reset as the device starts.  | |
```

Comment:
Reset to which value?

Reply:
Ah, I see that you are just calling `resetOrientation(XRM_Alignment)`, probably it is worth to mention this in the docs to specify what you meant with `reset`.